src/main/java/com/playonlinux/ui/impl/javafx/mainwindow/console/CommandHistory.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2015 Markus Ebner
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.playonlinux.ui.impl.javafx.mainwindow.console;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Small helper intended for keeping track of commands that have been entered into the console window.
+ */
+public class CommandHistory {
```

Comment:
`CommandHistory` can be moved, yes (last sentence of the pull-request message). If you tell me where to move it, I will do so and add the tests.